pages/osx/goku.md

```diff
@@ -0,0 +1,28 @@
+# goku
+
+> A tool to manage Karabiner configuration.
+> More information: <https://github.com/yqrashawn/GokuRakuJoudo>.
+
+- Generate karabiner.json by using default configuration:
+
+`goku`
+
+- Generate karabiner.json by using the specific config.edn:
+
+`goku --config {{path/to/config.edn}}`
+
+- Dry run new configuration into stdout instead of updating karabiner.json:
+
+`goku --dry-run`
+
+- Dry run whole configuration into stdout instead of updating karabiner.json:
+
+`goku --dry-run-all`
+
+- Show help:
+
+`goku --help`
+
+- Show version:
+
+`goku --version`
```

Comment:
```suggestion
> Manage Karabiner configuration.
> More information: <https://github.com/yqrashawn/GokuRakuJoudo>.

- Generate `karabiner.json` by using the default configuration:

`goku`

- Generate `karabiner.json` by using the specific `config.edn` file:

`goku --config {{path/to/config.edn}}`

- Dry run new configuration into `stdout` instead of updating `karabiner.json`:

`goku --dry-run`

- Dry run the whole configuration into `stdout` instead of updating `karabiner.json`:

`goku --dry-run-all`

- Display help:

`goku --help`

- Display version:

`goku --version`
```